configs/.env.devnet

```diff
@@ -1,5 +1,7 @@
-BOOT_NODE_IP_ADDRESS=164.90.173.129
-NETWORK_IDENTIFIER=12D3KooWFNt9SC7HruXFfdqspMtmRvq1n1dwi3FA5XHcs9ri3unt
+BOOT_NODE_IP_ADDRESS_1=164.90.173.129
```

Comment:
Keep this one without the suffix. We use it in different places already. Please, update here and in all other places.

Reply:
Updated.